doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -33,38 +67,105 @@ Matching FQN in a different Tenant | N/A | OK | OK
 Matching FQN at Application level | error | error | error unless the version is unique
 Matching FQN in a different Application of the same Tenant | N/A | N/A | warning in web UI; OK at services level
 
+## CT management ##
 
+CTs can be managed via [REST API]({{root_url}}Programming-guide/Server-REST-APIs#) calls or via Administration UI. Management of CTs in Administration UI is separated according to CT scope.
 
-Users are able to promote the CT's scope from APPLICATION to TENANT, provided that there is no other CT with the identical FQN within the given tenant, and that they have a permission to do so. Otherwise an appropriate error will be displayed.
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" href="#managing-system-ctl">Managing System CTL</a></li>
+  <li><a data-toggle="tab" href="#managing-tenant-ctl">Managing Tenant CTL</a></li>
+  <li><a data-toggle="tab" href="#managing-application-ctl">Managing Application CTL</a></li>
+</ul>
```

Comment:
There is quite a lot of duplication across these tabs. Can you move everything common out of tabs and only have discrepancies described in tabs?